how-not-to-be-a-dinosaur.md

```diff
@@ -0,0 +1,117 @@
+
+## Dem feelings...
+
+Who had I been kidding all these years calling myself a senior developer? How emberrassing was it that I had to ask my 12 years younger colleague for help over and over again...  
+
+![](https://raw.githubusercontent.com/MyOnlineStore/blogs/Dinosaurusblog/public/johan_knowsnothing.png)
+
+Some people will think "what's the big deal?", and I am happy for you if you do, but you have to understand that asking help was a big issue for me. And a major cause for my burn out and personality disorder that came with it.
+
+So I did two things: I decided to start learning all these new tools and code, but more importantly I started to ask for help. And the best thing I did: I told my colleagues how I was feeling and how insecure I was about everything. They responded very well to this and this gave me the confidence to go for it.
+
+Also, I realised I did not have to start all over again. It might have felt like it, but I still had a big headstart with the years of experience in programming overall. It's just a new language and new tools I needed to learn, not learning how to speak and not the first tool I ever used.
+
+It will take some time to learn all these new things. And that's okay. Good things take time. I should have realised this earlier since it's litteraly written on my body, but sadly it's not that obvious sometimes.
+
+![](https://raw.githubusercontent.com/MyOnlineStore/blogs/Dinosaurusblog/public/johan_arm.png)
+
+_Yes, this is also me._
+
+## … okay great, no fucks given, but what then?
+
+First I made a list of all the tools we use. It was quite a list with Typescript, Babel, Webpack, React, Apollo, Redux, GraphQL, Yarn and many more. This list was quite overwhelming, but then I started to put them into two groups: things I needed to learn right from the start and things I can figure out later. Turns out the first list is not that big and seemed do-able.
+
+I now know how to set up a project using Babel, but I have no idea about the inner workings tho. And that's okay. I will learn it some day. Until then it's perfecgtly fine to ask a colleague who does get it.
+
+Bottomline is that it is okay to not know things. It's not the amount of things you know that make you a senior developer, it's the way you handle things[that make you a senior](https://medium.com/better-programming/how-to-be-a-good-senior-developer-958948e02ada). In my opinion.
+
+I have always used to take some time for personal growth once in a while, but this was only a few spare hours now and then, but since a few months our company is doing so called "Ship-it" days where we developers can do whatever we want as long as it's somehow work related. This is a great way for me to learn stuff I might not naturally learn during everyday work. When I work in our normal codebase I just run `yarn dev` and storybook starts and that's it. But on these ship-it days I make new repo's, install packages and set them up. That is a great way to learn new things.
```

Comment:
```suggestion
I have always tried to take some time for personal growth once in a while, but this was only a few spare hours now and then. However, since a few months our company is doing so called "Ship-it" days where we developers can do whatever we want as long as it's somehow work related. This is a great way for me to learn stuff I might not naturally learn during everyday work. When I work in our normal codebase I just run `yarn dev`, storybook starts and that's it. But on these ship-it days I create new repos, install packages and set them up. That is a great way to learn new things.
```